syntax/storm.vim

```diff
@@ -22,45 +22,51 @@ syn keyword stormConditional switch if elif else try catch
 "syn region stormExprStmt start="\\$\\(" end="\\)"
 
 " Various operators
-syn match stormOperator "\V="
-syn match stormOperator "\V@="
-syn match stormOperator "\V*="
-syn match stormOperator "\V^="
-syn match stormOperator "\V!="
-syn match stormOperator "\V?="
-syn match stormOperator "\V>="
-syn match stormOperator "\V<="
-syn match stormOperator "\V~="
-syn match stormOperator "\V>"
-syn match stormOperator "\V<"
-syn match stormOperator "\V->"
-syn match stormOperator "\V<-"
-syn match stormOperator "\V-+>"
-syn match stormOperator "\V<+-"
-syn match stormOperator "\V*in="
-syn match stormOperator "\V+"
-syn match stormOperator "\V-"
-syn match stormOperator "\V*"
-syn match stormOperator "\V/"
-syn match stormOperator "\V&"
-syn match stormOperator "\V%"
-syn match stormOperator "\V|"
-" TODO: There's some array opers we might be missing
+syn match stormOperator "\V=" containedIn=stormEditParens
+syn match stormOperator "\V@=" containedIn=stormEditParens
+syn match stormOperator "\V*=" containedIn=stormEditParens
+syn match stormOperator "\V^=" containedIn=stormEditParens
+syn match stormOperator "\V!=" containedIn=stormEditParens
+syn match stormOperator "\V?=" containedIn=stormEditParens
+syn match stormOperator "\V>=" containedIn=stormEditParens
+syn match stormOperator "\V<=" containedIn=stormEditParens
+syn match stormOperator "\V~=" containedIn=stormEditParens
+syn match stormOperator "\V>" containedIn=stormEditParens
+syn match stormOperator "\V<" containedIn=stormEditParens
+syn match stormOperator "\V->" containedIn=stormEditParens
+syn match stormOperator "\V<-" containedIn=stormEditParens
+syn match stormOperator "\V-+>" containedIn=stormEditParens
+syn match stormOperator "\V<+-" containedIn=stormEditParens
+syn match stormOperator "\V*in=" containedIn=stormEditParens
+syn match stormOperator "\V+" containedIn=stormEditParens
+syn match stormOperator "\V-" containedIn=stormEditParens
+syn match stormOperator "\V*" containedIn=stormEditParens
+syn match stormOperator "\V/" containedIn=stormEditParens
+syn match stormOperator "\V&" containedIn=stormEditParens
+syn match stormOperator "\V%" containedIn=stormEditParens
+syn match stormOperator "\V|" containedIn=stormEditParens
+syn match stormOperator "\V*near=" containedIn=stormEditParens
+syn match stormOperator "\V*unset=" containedIn=stormEditParens
```

Comment:
There's also a `*$foo=` where foo could be `unset`, `always`, `never`. Not sure if you can encode that here.